code/game/objects/buckling.dm

```diff
@@ -5,6 +5,10 @@
 	var/buckle_lying = -1 //bed-like behaviour, forces mob.lying = buckle_lying if != -1
 	var/buckle_requires_restraints = 0 //require people to be handcuffed before being able to buckle. eg: pipes
 	var/list/mob/living/buckled_mobs = null //list()
+	var/friend_buckle = TRUE //if someone else can buckle them
+	var/friend_unbuckle = TRUE //if someone else can unbuckle them
+	var/self_buckle = TRUE //if someone can buckle themselves
+	var/self_unbuckle = TRUE //if someone can unbuckle themselves
```

Comment:
Please explain their purpose and whether or not they're relative to the object buckled onto or the mob being buckled.

Reply:
yes, please make this a flag controlled var.

If you were super dooper cool you would also roll buckle_requires_restraint into this new flag

call it buckle_requirements or buckle_restrictions

BUCKLE_FRIEND
UNBUCKLE_FRIEND
BUCKLE_SELF
UNBUCKLE_SELF

etc

Reply:
https://tgstation13.org/wiki/Binary_flags

Reply:
yep lets do it